docs/targets/solana.rst

```diff
@@ -23,60 +23,96 @@ This is how to build your Solidity for Solana:
 
   solang compile --target solana flipper.sol -v
 
-This will produce two files called `flipper.abi` and `flipper.so`. The former is an ethereum style abi file and the latter is
+This will produce two files called `flipper.json` and `flipper.so`. The former is an Anchor style abi file and the latter is
 the ELF BPF shared object which can be deployed on Solana. For each contract, Solang will create an ABI file and a binary file
-`contract-name.so`, which contains the code.
+`contract-name.so`, which contains the program.
+
+The program will need to be deployed. Usually, the program is deployed at a well-known address which is specified in the Solidity
+source code using the `@program_id("F1ipperKF9EfD821ZbbYjS319LXYiBmjhzkkf5a26rC")` annotation on the contract.
+In order to deploy to this address, you'll need its the private key. For the flipper example, we
+have included the private key in `flipper-keypair.json`, so you can use that here for deploying the program:
 
 .. code-block:: bash
 
-    npm install @solana/solidity
+    solana program deploy --program-id flipper-keypair.json flipper.so
+
+This is of course an example. Never save private keys for your program to any public repository, this is of course a security
```

Comment:
We should not have this in our docs. Users will adapt whatever we write here inevitably, regardless of how much disclaimers we put up

Reply:
Sure, no problem with leaving some disclaimers. This comment goes in combination with my suggestion below.

Reply:
I've removed the disclaimer. I do agree we should lead by example and not "do as a say, not as a I do".

In the future (hopefully) `anchor init` will generate a private key and set it in the source code and we can remove all this.